schema/course.model.schema

```diff
@@ -531,6 +531,29 @@
         }
       }
     },
+    "_navigation": {
+      "type": "object",
+      "title": "Navigation settings",
+      "properties" : {
+        "_isDefaultNavigationDisabled": {
+          "type": "boolean",
+          "default": false,
+          "required": true,
+          "inputType": "Checkbox",
+          "validators": [],
+          "title": "Disable default navigation bar?"
+        },
+        "_bottomOnSelector": {
```

Comment:
`_bottomOnSelector` is a little confusing as an name

Perhaps an alternative way could be to change the naming to something similar to `_navigationRenderArea` and utilise inputType, with a default, similar to https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/properties.schema#L758

Reply:
`_navigationAlignment: 'top', 'bottom', 'bottom-on-mobile'` ?